charts/images.yaml

```diff
@@ -191,9 +191,9 @@ images:
   tag: "v1.2.0"
   targetVersion: ">= 1.22"
 - name: ingress-default-backend
-  sourceRepository: github.com/gardener/ingress-default-backend
-  repository: eu.gcr.io/gardener-project/gardener/ingress-default-backend
-  tag: "0.10.0"
+  sourceRepository: https://github.com/kubernetes/ingress-gce
+  repository: k8s.gcr.io/defaultbackend-amd64
+  tag: "1.5"
```

Comment:
I recall in the past we had issues with the internal CI/CD version handling if the version tag is not semver. For example for alicloud-csi-driver we had to use custom repository with a semver tag and I recall I had to ask the upstream folks to fix their tags to be semver (ref https://github.com/kubernetes-sigs/alibaba-cloud-csi-driver/issues/444). 
`1.5` does not look like a valid semver. Can we check with @ccwienk whether such version can cause again similar problems?

Reply:
I asked them.

Reply:
I now see that we have other images in the same format (`<major>.<minor>`). So, I think we can assume it is supported.

https://github.com/gardener/gardener/blob/a59e5dbfa51d3ddee06aeb5b0a76182d519425d4/charts/images.yaml#L19-L22

Reply:
Right, thanks for checking!